examples/tls-termination/haproxy/ingress-tls-default.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: app
+spec:
+  tls:
+  - hosts:
+    - foo.bar
+  rules:
+  - host: foo.bar
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: nginx
```

Comment:
I'd like to see here something easy to start with. This sample would be a bit complicated: the user need to open another page (`prereq.md`) to discover he need to open another one to find a resource (`http-svc.yaml`) he need to deploy, something he could simply create in a single line.

What about reuse the same svc and image of the prereq but add a `kubectl run --expose` here, removing one prereq?